### DISCUSSION TITLE
selectOnTab and advance to next field

### DISCUSSION THREAD

**Discussion by powerbuoy** [orchidjs/tom-select#111] [Ideas]
I'd like to use `selectOnTab: true` but still advance to the next form field after selection. This makes a lot of sense for single value selects as it's exactly how a normal select element works.

Perhaps an option to disable this behaviour would be enough? https://github.com/orchidjs/tom-select/blob/master/src/tom-select.ts#L679

**Comment by oyejorge** [comment]
Looks like there's a bug with the current tab handing that is related to this. I'll have a fix out shortly

**Reply by powerbuoy** [reply]
Awesome, thanks!

**Reply by powerbuoy** [reply]
Any progress on this? I just updated to 1.7.5 and hitting tab still keeps focus in the same field.

**Reply by oyejorge** [reply]
The only thing I can see that's different between native `<select>` and a Tom Select instance is that the dropdown opens when tabbing into a Tom Select instance. And that can be turned off with `openOnFocus = false`. Does that solve things for you?

**Reply by powerbuoy** [reply]
Not really, what I'm after is being able to tab into a tom-select, then arrow key to my choice, tab to select _and_ go to the next field.

You're right that select elements don't work that way though (at least not on Chrome / Mac) but an input with a datalist does: https://codepen.io/powerbuoy/pen/qBreOWm

If you

1. Focus the "Hello" field
2. Tab
3. Arrow key
4. Tab
5. You'll end up on the "Good bye" field

**Reply by mirrorspock** [reply]
I ran into the same issue, a single select, which gets picket with tab via selectOnTab doesn't then immediately tab to the next field., 

so you enter the select, it opens, you type and arrow to the option, press tab to select it, then tab to go to the next field.

**Comment by ham17ahm** [comment]
Any updates on this? Has this option been added yet?
